lite/fn.cpp

```diff
@@ -7337,6 +7337,7 @@ str_to_lower(str, lcstr);													// 09/28/00 AM.
 _TCHAR *istr = 0;
 
 long pos;
+_TCHAR buf2[1001];
```

Comment:
Needed to be moved out of the scope because theoretically the memory can be reused once the scope is left.